signer/src/request_decider.rs

```diff
@@ -248,6 +276,20 @@ where
         Ok(())
     }
 
+    /// Send the given deposit decisions to the other signers for redundancy.
+    #[tracing::instrument(skip_all)]
+    pub async fn handle_deposit_decisions_to_retry(
+        &mut self,
+        decisions: Vec<model::DepositSigner>,
+        chain_tip: &BitcoinBlockHash,
+    ) -> Result<(), Error> {
+        for decision in decisions.into_iter().map(SignerDepositDecision::from) {
+            self.send_message(decision, chain_tip).await?;
```

Comment:
Since this retry is a "best effort" type of thing, should we just log the error in here instead of bailing at the first one?